core/services/relay/evm/chain_reader_test.go

```diff
@@ -2,6 +2,7 @@ package evm_test
 
 import (
```

Comment:
I don't really understand what is the point of adding CW into EVM CR tester? CW should be tested separately from CR